operator/src/main/resources/scripts/utils.sh

```diff
@@ -889,3 +889,94 @@ function adjustPath() {
     fi
   fi
 }
+
+#
+# checkCommonMount
+#   purpose: If the COMMON_MOUNT_PATH directory exists, it echoes the contents of output files
+#            in ${COMMON_MOUNT_PATH}/commonMountLogs dir. It returns 1 if a SEVERE message
+#            is found in any of the output files in ${COMMON_MOUNT_PATH}/commonMountLogs dirs.
+#            It also returns 1 if 'successfully' message is not found in the output files
+#            or if the COMMON_MOUNT_PATH directory is empty. Otherwise it returns 0 (success).
+#            See also 'commonMount.sh'.
+#
+function checkCommonMount() {
+  # check commonMount results (if any)
+  if [ -z "$COMMON_MOUNT_PATHS" ]; then
+    trace FINE "Common Mount: Skipping common mount checks (no common mounts configured)."
+    return
+  fi
+
+  trace FINE "Common Mount: COMMON_MOUNT_PATHS is '$COMMON_MOUNT_PATHS'."
+  for COMMON_MOUNT_PATH in ${COMMON_MOUNT_PATHS/,/ }; do
+    trace FINE "Common Mount: COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH'."
+    traceDirs $COMMON_MOUNT_PATH
+    touch ${COMMON_MOUNT_PATH}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "Common Mount: Cannot write to the COMMON_MOUNT_PATH '${COMMON_MOUNT_PATH}'. " \
+                   "This path is configurable using the domain resource 'spec.commonMountVolumes.mountPath' " \
+                   "attribute." && return 1
+    fi
+    rm -f ${COMMON_MOUNT_PATH}/testaccess.tmp || return 1
+
+    out_files=$(set -o pipefail ; ls -1 $COMMON_MOUNT_PATH/commonMountLogs/*.out 2>1 | sort --version-sort) \
+      || (trace SEVERE "Common Mount: Assertion failure. No files found in '$COMMON_MOUNT_PATH/commonMountLogs/*.out" \
+      && return 1)
+    severe_found=false
+    for out_file in $out_files; do
+      if [ "$(grep -c SEVERE $out_file)" != "0" ]; then
+        trace SEVERE "Common Mount: Error found in file '${out_file}' while initializing commonMount."
+        severe_found=true
+      elif [ "$(grep -c successfully $out_file)" = "0" ]; then
+        trace SEVERE "Common Mount: Command execution was unsuccessful in file '${out_file}' while initializing commonMount."
+        severe_found=true
```

Comment:
If we arrive here, the `trace "Common Mount: Contents of '${out_file}':"` should be skipped, so that the operator's parse of this SEVERE will include the contents of the .out file.  In addition, add the sentence 'Contents of '${out_file}' to this SEVERE.